features/custom-user-agent.json

```diff
@@ -0,0 +1,19 @@
+{
+    "_meta": {
+        "description": "Custom user agents used by our native apps that override the default user-agent string.",
+        "sampleExcludeRecords": {
+            "domain": "example.com",
+            "reason": "Two factor auth that verifies device pathes using user agent"
+        }
+    },
+    "settings": {
+        "omitApplicationSites": [
```

Comment:
Can't these be regular exceptions?

Reply:
They _could_. However given how they're used (selectively dropping parts of the custom UA) I don't think its appropriate. As mentioned below Android has a more granular approach. IMO exceptions on this feature should be used if _no_ UserAgent customization should happen at all.